test/e2e/framework/framework.go

```diff
@@ -202,7 +202,8 @@ func (f *Framework) BeforeEach() {
 	// https://github.com/onsi/ginkgo/issues/222
 	f.cleanupHandle = AddCleanupAction(f.AfterEach)
 
-	if f.Client == nil {
+	// Create an OCI client if the cloudConfig has been specified.
```

Comment:
A series of tweaks to allow users to avoid specifying the cloud config if not required.